styles/properties.less

```diff
@@ -100,7 +102,7 @@
     }
   }
 
-  button {
+  .action-button {
```

Comment:
Should we also indicate this as breaking change?

Reply:
I believe this is a breaking change for anyone using custom entries. I will have to change my code in order to make buttons look OK again.

Reply:
Unfortunately, it was necessary. `button { position: absolute; }` is just unacceptable. I will add a note about the breaking change to the respective commit.